domain/src/main/java/com/example/domain/repository/ApolloCountryRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.domain.repository
+
+import com.example.domain.model.Post
+
+interface ApolloCountryRepository {
```

Comment:
`Apollo` has nothing to do with this GraphQL API. We should name our classes/files/functions better. For example, this file and interface should only be named `CountryRepository`.